mautic/mautic
# Contact Stages

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | N
| New feature?  | Y 
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  n/a

## Description
This PR adds a new Stages bundle that will be used to set a Contact's stage.  A contact can now move from stage to stage with different actions like if a contact: visits a landing page, is sent an email, opens an email, submits a form, or downloads an asset.
This can be configured through the stages menu or through a campaign.

- Working on having a way that contacts move stages based on points reached, or when an sms is sent. 
- Also working on a way to add a weight to stages so that contacts don't move back on stages unless specified.

## Steps to test this PR
Apply this Pull request, create the set of stages ie.  Contacted (when an email is sent), Engaged (when a contact visits a landing page) Returning (when a contact visits a page more than once).

Contact should move from stage to stage.
